<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionBackup"/>
   <addaction name="actionRestore"/>
   <addaction name="separator"/>
   <addaction name="actionData_Tree"/>
   <addaction name="actionHighlight_Differences"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionHighlight_Differences">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight Differences</string>
   </property>
   <property name="toolTip">
    <string>Highlight differences between the source and the selected backup in the data-tree (may take some time)</string>
   </property>
  </action>
  <action name="actionData_Tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data Tree</string>
   </property>
   <property name="toolTip">
    <string>Show files in Backup as a tree. This may take some time for large Backups</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
